test/extendedAdvertisement.js

```diff
@@ -77,15 +80,15 @@ contract('ExtendedAdvertisement', function(accounts) {
 
   		await appcInstance.approve(addInstance.address,campaignBudget);
 
-  		await addInstance.createCampaign(packageName,countryList,[1,2],campaignPrice,campaignPrice,startDate,endDate, "appcoins.io");
+  		await addInstance.createCampaign(packageName,countryList,[1,2],campaignPrice,campaignPrice,startDate,endDate, accounts[8],"appcoins.io");
```

Comment:
We could assign accounts[8] and "appcoins.io" to variables. But that's not mandatory.